sdk.yaml

```diff
@@ -286,6 +286,13 @@ Realtime:
       .synopsis: |
         Override the default value using the `suspendedRetryTimeout` client option.
         Reconnection is attempted after this amount of time in the `SUSPENDED` connection state.
+    OS Connectivity Events:
+      .specification: [RTN20]
+      .synopsis: |
+        Listen to Operating System network connectivity events
+        When `CONNECTED`, `CONNECTING`, or `DISCONNECTING` and the underlying internet connection is no longer available, the connection should transition to `DISCONNECTED`.
```

Comment:
Is it worth linking this to RTN15 as we do in the other specification? :)

Reply:
good idea, done!